tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.cc

```diff
@@ -303,27 +304,29 @@ void EnergyPlusFixture::TearDown()
 void EnergyPlusFixture::clear_all_states()
 {
     // A to Z order
+    AirflowNetwork::clear_state();
     AirflowNetworkBalanceManager::clear_state();
     BaseboardElectric::clear_state();
     BaseboardRadiator::clear_state();
     Boilers::clear_state();
     BoilerSteam::clear_state();
     BranchInputManager::clear_state();
-    CoolingPanelSimple::clear_state();
     ChillerAbsorption::clear_state();
     ChillerElectricEIR::clear_state();
     ChillerExhaustAbsorption::clear_state();
     ChillerGasAbsorption::clear_state();
     ChillerIndirectAbsorption::clear_state();
+    clearCoilSelectionReportObj(); // ReportCoilSelection
+    clearFacilityElectricPowerServiceObject();
```

Comment:
Thanks for cleaning up the alphabetical order.  I really don't like these specialized functions being called, it would be really nice if these clear functions were simply included in their appropriate module's clear_state functions.  I know this is not something you added, so don't make any changes here, I'm just lamenting :laughing: